huaweicloud/resource_huaweicloud_identity_group_membership.go

```diff
@@ -65,12 +67,7 @@ func resourceIdentityGroupMembershipV3Read(d *schema.ResourceData, meta interfac
 
 	allPages, err := users.ListInGroup(identityClient, group, users.ListOpts{}).AllPages()
 	if err != nil {
-		if _, b := err.(golangsdk.ErrDefault404); b {
-			d.SetId("")
-			return nil
-		} else {
-			return fmtp.Errorf("Unable to query groups: %s", err)
-		}
+		return common.CheckDeleted(d, err, "Unable to query groups")
```

Comment:
CheckDeleted will wrap the 404 error message with your specifed one ("%s: %s", your_msg_above, err), so here better to use something lie `identity_group_memeber` to specify which resource logs this error.